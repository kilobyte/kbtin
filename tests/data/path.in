#act rep {#path;#show $PATH;#ret all;#map s;#map e;#map s;#path;#ret;#ret;#del 1 #end}
#pathd n s
#pathd s n
#run a //selfpipe
n
n
s
rep
