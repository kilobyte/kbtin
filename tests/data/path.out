n
n
s
#Path:  n;n;s
n n s
#Path:  s;s
rep
n
s
s
n
n

#SESSION 'a' DIED.
